docs/spec/relayer/Relayer.tla

```diff
@@ -5,111 +5,31 @@
  It instantiates the module Environment, which takes care of the chain logic. 
```

Comment:
ICS18Environment instead of Environment.

Reply:
I think the "comment header" of the Relayer module is outdated. The Relayer module no longer instantiates the Environment: it's the other way around now, so we should probably update this comment to reflect the new situation.